core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -81,7 +89,38 @@ private[spark] class DiskBlockManager(blockManager: BlockManager, conf: SparkCon
     new File(subDir, filename)
   }
 
-  def getFile(blockId: BlockId): File = getFile(blockId.name)
+  /** Looks up a file by blockManagerId */
+  // This method should be kept in sync with
+  // org.apache.spark.network.shuffle.StandaloneShuffleBlockManager#getFile().
```

Comment:
It seems that the changes here caused this to be out of sync, right? Do we want to make the same changes in `StandaloneShuffleBlockManager` too?